Components/Pages/SignUpPassword.js

```diff
@@ -20,7 +20,10 @@ const SignUpPassword = ({ navigation }) => {
         password,
         userName,
       })
-      .then(response => AsyncStorage.setItem("token", response["data"]["token"]))
+      .then(response => {
+        AsyncStorage.setItem("token_created_date", moment().format())
```

Comment:
@ChrisCruze moment is not imported here
it is throwing this error when signing up: **Object {
  "error": [ReferenceError: Can't find variable: moment],
}**

Reply:
done - thanks for catching that